assets/css/style.css

```diff
@@ -46,4 +87,50 @@ body {
     line-height: 25px;
     letter-spacing: 0.025em;
     color: var(--white-900);
-}
\ No newline at end of file
+    cursor: pointer;
+}
+
+.nav__list__item:not(:last-child) {
+    margin-right: 70px;
+}
+
+
+/* /header */
+
+
+/* main */
+
+.main {
+    height: 100vh;
+    padding: 223px 368px 153px;
+    background-image: url(/assets/img/headerIMG.jpg);
```

Comment:
delete `/assets/img/headerIMG.jpg`
it should be like this `./../img/headerIMG.jpg`